.golangci.yaml

```diff
@@ -0,0 +1,11 @@
+linters:
+  enable:
+  - errcheck
+  - exhaustive
+  - gosimple
+  - govet
+  - ineffassign
+  - misspell
+  - staticcheck
+  - typecheck
+  - unused
```

Comment:
@flavianmissi do you want to add something from https://golangci-lint.run/usage/linters/ ?

Reply:
let's enable it too